api/src/controllers/collection-controller.js

```diff
@@ -101,19 +101,18 @@ const updateCollection = async function(incomingObj, collectionId, displayName)
     let recordsToAdd = [];
     let arrayOfObjIds = [];
 
-    for (let i = 0; i < incomingObj.records.length; i++) {
-      if (!recordIds.includes(incomingObj.records[i])) {
-        // These are new records that need to be added to the collection.
-        recordsToAdd.push(incomingObj.records[i]);
+    for(const record of incomingObj.records){
+      if(recordIds.includes(record)){
```

Comment:
Was the `!` operator meant to be removed here?

Reply:
That was an oversight, thank you for catching that!